index.html

```diff
@@ -155,70 +155,73 @@ <h3 class="project-title">Multi-Post Stories</h3>
       </section> <!-- .works -->
       
       <section class="about-myself">
-        <h2 class="about">About<br>Myself</h2>
-        <p class="introduction"> 
-            Hello, I'm a software developer! I can help you build a product,
-           feature or website Look through some of my work and experience! 
-           If you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <a href="https://linkedin.com/in/OscarFM-dev" class="connect-text">LET'S CONNECT</a>
-        <ul class="social-media-icons">
-          <li>
-            <a href="https://linkedin.com/in/OscarFM-dev">
-              <img src="./images/header-section/linkedin.png" alt="linkedin logo" class="linkedin-logo">
-            </a>
-          </li>
-          <li>
-            <a href="https://github.com/OscarFM-dev">
-              <img src="./images/header-section/github.svg" alt="github logo">
-            </a>
-          </li>
-          <li>
-            <a href="https://twitter.com/OscarFMdev">
-              <img src="./images/header-section/twitter.svg" alt="twitter logo">
-            </a>
-          </li>
-          <li>
-            <a href="https://medium.com/@oscarfernandezm95">
-              <img src="./images/header-section/medium.svg" alt="medium logo">
-            </a>
-          </li>
-          <li>
-            <a href="https://www.instagram.com/oscarfmdev/">
-              <img src="./images/header-section/peace.svg" alt="peace logo">
-            </a>
-          </li>
-        </ul>
-        <button class="resume-button">Get my resume</button>
-
-        <div class="languages-section">
-          <h4 class="languages">Languages</h4>
-          <img class="arrow" src="./images/arrows/down-arrow.svg" alt="down-arrow">
+        <div class="about-first">
+          <h2 class="about">About Myself</h2>
+          <p class="introduction"> 
+              Hello, I'm a software developer! I can help you build a product,
+             feature or website Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don't hesitate to contact me.
+          </p>
+          <a href="https://linkedin.com/in/OscarFM-dev" class="connect-text">LET'S CONNECT</a>
```

Comment:
- [ ] please reduce the size of the `lets connect` social icons to match that of figma. align the `JavaScript` logo text with the languages and framework, And kindly add the size of `get my resume` button and align it with the text as well.
figma
![Screenshot (337)](https://user-images.githubusercontent.com/65298048/186539471-2ed4eacc-706e-4092-9d6e-79aa9843e5fa.png)
yours
![Screenshot (341)_LI](https://user-images.githubusercontent.com/65298048/186539297-4efcfbfb-4d2d-4ac0-8e09-38d34b720080.jpg)

Reply:
Good job following the advice of the previous reviewer